src/index.js

```diff
@@ -1,3 +1,3 @@
 export Grid from './components/Grid';
-export Row from './components/Row';
-export Col from './components/Col';
+export * from './components/Row';
+export * from './components/Col';
```

Comment:
I would rather if these exports were explicit, just so we don't accidentally expose some internal API in the future. Combined with leaving `Col` and `Row` as default exports, I think this should be:

```js
export Row, { getRowClassNames } from './components/Row';
export Col, { getColClassNames } from './components/Col';
```

Not sure if this is valid syntax, though. 😅